addons/account/models/account_move.py

```diff
@@ -361,7 +361,7 @@ def _get_counterpart(self):
             counterpart = list(counterpart)[0:2] + ["..."]
         self.counterpart = ",".join(counterpart)
 
-    name = fields.Char(required=True, string="Label")
+    name = fields.Char(string="Label")
```

Comment:
Ok, I got the point, but I continue thinking either the ref and/or the name must be mandatory, it does not make much sense have them both not mandatory and then start to have hundreds of lines without any explanation.

I mean, The name or the ref must be mandatory, if the user copy and paste the text it is his problem, our responsibility is until say "Hey! you must say why are you putting this number here".

I hope it explain better my point.